pkg/io/jsonio/unmarshal.go

```diff
@@ -28,6 +29,12 @@ func (*Reader) ReadSpec(filename string, configDefs *ir.ConfigDefs) (*ir.Spec, e
 		return nil, err
 	}
 
+	cidrSegments := translateSegments(jsonspec.Segments, TypeCidr)
+	finalCidrSegments, err := computeContainedSubentsInCidrSegment(configDefs, cidrSegments)
```

Comment:
Move the call to `translateSegments()` inside `computeContainedSubentsInCidrSegment()`. It will not change the number of arguments, and will avoid having a temporary local variable that is not needed anywhere else.

Reply:
On a second thought, there's better way to do that